test/cql-pytest/test_tablets.py

```diff
@@ -121,3 +121,49 @@ def test_tablets_are_dropped_when_dropping_table(cql, skip_without_tablets):
             assert res.one().tablet_count > 0, f"table {keyspace_name}.{table_name}: zero tablets allocated"
         res = cql.execute(f"SELECT * FROM system.tablets WHERE keyspace_name='{keyspace_name}' AND table_name='{table_name}' ALLOW FILTERING")
         assert not res, f"table {keyspace_name}.{table_name} was found in system.tablets after the table was dropped: tablet_count={res.one().tablet_count}"
+
+
+def test_tablets_are_dropped_when_dropping_view(cql, skip_without_tablets):
+    ksdef = "WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'replication_factor' : '1'} AND TABLETS = {'initial': 1}"
+    with new_test_keyspace(cql, ksdef) as keyspace:
```

Comment:
Nitpick:
You don't have to create a new keyspace in all these tests - this creation slows down the test and makes it longer and harder to read (and more indented). You can just use the same "test_keyspace" fixture in all these tests. It will use tablets.